server/src/main/java/org/elasticsearch/action/admin/cluster/stats/ClusterStatsResponse.java

```diff
@@ -149,4 +160,117 @@ public String toString() {
+            if (remoteResponse != null) {
+                this.clusterUUID = remoteResponse.getClusterUUID();
+                this.versions = remoteResponse.getVersions();
+                this.status = remoteResponse.getStatus().name().toLowerCase(Locale.ROOT);
+                this.nodesCount = remoteResponse.getNodesCount();
+                this.shardsCount = remoteResponse.getShardsCount();
+                this.indicesCount = remoteResponse.getIndicesCount();
+                this.indicesBytes = remoteResponse.getIndicesBytes();
+                this.heapBytes = remoteResponse.getHeapBytes();
+                this.memBytes = remoteResponse.getMemBytes();
+            } else {
+                this.status = "unavailable";
+                this.clusterUUID = "unavailable";
+                this.versions = Set.of();
+                this.nodesCount = 0;
+                this.shardsCount = 0;
+                this.indicesCount = 0;
+                this.indicesBytes = 0;
+                this.heapBytes = 0;
+                this.memBytes = 0;
+            }
+        }
+
+        public String getClusterUUID() {
+            return clusterUUID;
+        }
+
+        public String getMode() {
+            return mode;
+        }
+
+        public boolean isSkipUnavailable() {
+            return skipUnavailable;
+        }
+
+        public String getTransportCompress() {
+            return transportCompress;
+        }
+
+        public Set<String> getVersions() {
+            return versions;
+        }
+
+        public String getStatus() {
+            return status;
+        }
+
+        public long getNodesCount() {
+            return nodesCount;
+        }
+
+        public long getShardsCount() {
+            return shardsCount;
+        }
+
+        public long getIndicesCount() {
+            return indicesCount;
+        }
+
+        public long getIndicesBytes() {
+            return indicesBytes;
+        }
+
+        public long getHeapBytes() {
+            return heapBytes;
+        }
+
+        public long getMemBytes() {
+            return memBytes;
+        }
+
+        @Override
+        public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+            builder.startObject();
+            builder.field("cluster_uuid", clusterUUID);
+            builder.field("mode", mode);
+            builder.field("skip_unavailable", skipUnavailable);
+            builder.field("transport.compress", transportCompress);
+            builder.field("status", status);
+            builder.field("version", versions);
+            builder.field("nodes_count", nodesCount);
+            builder.field("shards_count", shardsCount);
+            builder.field("indices_count", indicesCount);
+            builder.field("indices_total_size_bytes", indicesBytes);
+            builder.field("max_heap_bytes", heapBytes);
+            builder.field("mem_total_bytes", memBytes);
```

Comment:
Can we make these fields of type `ByteValue` and add support for the `?human` option using `builder.humanReadableField(...)`?

Reply:
It's intended mainly for Kibana consumption now, but I'll leave it to product again @naj-h what do you think?

Reply:
Discussed and resolved we will support human-readable options in accordance with how the rest of the values in stats are handled.